samples/bluetooth/bap_broadcast_sink/src/main.c

```diff
@@ -91,12 +91,17 @@ static const struct bt_audio_codec_cap codec_cap = BT_AUDIO_CODEC_CAP_LC3(
 	CONFIG_MAX_CODEC_FRAMES_PER_SDU,
 	(BT_AUDIO_CONTEXT_TYPE_CONVERSATIONAL | BT_AUDIO_CONTEXT_TYPE_MEDIA));
 
-/* Create a mask for the maximum BIS we can sync to using the number of streams
- * we have. Bit 0 is BIS index 1.
- */
-static const uint32_t bis_index_mask = BIT_MASK(CONFIG_BT_BAP_BROADCAST_SNK_STREAM_COUNT);
+struct bis_index_data {
+	uint32_t bis_index_bitfield[CONFIG_BT_BAP_BASS_MAX_SUBGROUPS];
+	enum bt_audio_location audio_allocation[BT_ISO_BIS_INDEX_MAX];
+	uint8_t num_subgroups;
+};
```

Comment:
Given that this is a sample, please provide comments for the purpose of this struct (ideally we should be quite verbose in terms of comments to help provide understanding in samples)